jsonnet/kube-prometheus/rules/general.libsonnet

```diff
@@ -0,0 +1,19 @@
+{
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: 'kube-prometheus-general.rules',
+        rules: [
+          {
+            expr: 'count (up == 1) without (instance, pod, node)',
```

Comment:
promql style best practices are having the `by` or `without` statement before the thing that we're actually counting, so flipping the statement to:

```
count without (instance, pod, node) (up == 1)
```